Source/FieldSolver/SpectralSolver/SpectralAlgorithms/PsatdAlgorithmMultiJ.H

```diff
@@ -0,0 +1,145 @@
+/* Copyright 2019
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef WARPX_PSATD_ALGORITHM_MULTIJ_H_
+#define WARPX_PSATD_ALGORITHM_MULTIJ_H_
+
+#include "FieldSolver/SpectralSolver/SpectralFieldData.H"
+#include "FieldSolver/SpectralSolver/SpectralKSpace.H"
+#include "SpectralBaseAlgorithm.H"
+
+#include <AMReX_Array.H>
+#include <AMReX_Config.H>
+#include <AMReX_REAL.H>
+
+#include <AMReX_BaseFwd.H>
+
+#include <array>
+#include <memory>
+
+#if WARPX_USE_PSATD
+/*
+ * \brief Class that updates the field in spectral space according to the multi-J algorithm
+ *        and stores the coefficients of the corresponding update equation.
```

Comment:
Could you add one or two lines here summarizing what is different between this algorithm and the regular PSATD algorithm. (e.g. you had that line in the previous version of the code: `two currents computed at the beginning and the end of the time interval instead of one current computed at half time`. I would also explicitly mention that J is assumed to be linear in time in this case)